src/css/components/_error.scss

```diff
@@ -1,48 +1,4 @@
-.vjs-error-display {
-  display: none;
-}
-
-.vjs-error .vjs-error-display {
-  display: block;
-  position: absolute;
-  left: 0;
-  top: 0;
-  width: 100%;
-  height: 100%;
-}
-
-.vjs-error .vjs-error-display:before {
-  content: 'X';
```

Comment:
was this removed on purpose?
I would prefer to not change the look of the error display *too* much. Or at least, we still need something to make it noticeable as an error display and not just an uncloseable modal dialog.